views/js/e2e/_helpers/commands/setupCommands.js

```diff
@@ -108,8 +108,12 @@ Cypress.Commands.add('setDeliveryForGuests', (testName) => {
     // Select example delivery
     cy.get(setupSelectors.deliveriesPage.rootDeliveryClass).contains(testName).click();
 
-    // Set guest access on the delivery
-    cy.get(setupSelectors.deliveriesPage.formContainer).contains('Guest Access').click();
+    // Set guest access on the delivery (if not yet enabled)
+    cy.get(setupSelectors.deliveriesPage.guestAccessCheckbox).then($checkbox => {
```

Comment:
I think a simple `cy.get(setupSelectors.deliveriesPage.guestAccessCheckbox).check();` will do the work.